www_src/pages/element/element.jsx

```diff
@@ -20,7 +19,10 @@ var testIds = {
 };
 
 render(React.createClass({
-  mixins: [router],
+  mixins: [
+    require('../../lib/router.jsx'),
+    require('../../lib/snackbar-mixin')
```

Comment:
if we put this into `Base`, that should add snackbar notifications for all views